res/values-ko/strings.xml

```diff
@@ -3,55 +3,55 @@
   <string name="app_name_release">Unexpected Keyboard</string>
   <!-- <string name="app_name_debug">Unexpected Keyboard (Debug)</string> -->
   <string name="short_description">개발자들을 위한 가벼운 가상 키보드.</string>
-  <!-- <string name="store_description">"The main feature is that you can type more characters by swiping the keys towards the corners.
+  <string name="store_description">이 애플리케이션의 주된 기능은 모서리로 키를 스와이프하는 것으로 더 많은 문자를 입력할 수 있는 것입니다.
 
-This application was originally designed for programmers using Termux.
-Now perfect for everyday use.
+이 애플리케이션은 원래 Termux를 사용하는 프로그래머를 위해 디자인되었습니다.
+Теперь идеально подходит для повседневного использования.
```

Comment:
What's this sentence written in cyrillic ? It translates weirdly.

Reply:
I think I made a mistake while doing something else. Sorry!